bucket/inkscape.json

```diff
@@ -32,25 +32,30 @@
             "$64bit_dl = Invoke-WebRequest ($redirUrl + '/windows/64-bit/compressed-7z/dl/') -UseBasicParsing",
             "$32bit_url = $32bit_dl.links | Where-Object href -match '\\.7z$' | Select-Object -first 1 -expand href",
             "$64bit_url = $64bit_dl.links | Where-Object href -match '\\.7z$' | Select-Object -first 1 -expand href",
-            "$inkscape_dir = ($64bit_url -split '/' | Select-Object -last 1) -split '-x64.7z' | Select-Object -first 1",
+            "$32bit_hash_url = $32bit_dl.links | Where-Object href -match '\\.7z_.+\\.sha256$' | Select-Object -first 1 -expand href",
+            "$64bit_hash_url = $64bit_dl.links | Where-Object href -match '\\.7z_.+\\.sha256$' | Select-Object -first 1 -expand href",
+            "$inkscape_dir = ($64bit_url -split '/' | Select-Object -last 1) -split '-x64_.+\\.7z' | Select-Object -first 1",
             "$scriptver = ($64bit_dl | Select-String -Pattern 'inkscape-([\\d.]+_[\\d-]+)_').Matches.Groups[1].Value",
-            "Write-Output $scriptver $32bit_url $64bit_url $inkscape_dir"
+            "Write-Output $scriptver $32bit_url $64bit_url $inkscape_dir $32bit_hash_url $64bit_hash_url"
         ],
-        "regex": "(?<version>[\\d.]+_[\\d-]+)\\s(?<win32biturl>.+)\\s(?<win64biturl>.+)\\s(?<inkscapedir>.+)"
+        "regex": "(?<version>[\\d.]+_[\\d-]+)\\s(?<win32biturl>.+)\\s(?<win64biturl>.+)\\s(?<inkscapedir>.+)\\s(?<win32bithashurl>.+)\\s(?<win64bithashurl>.+)"
     },
     "autoupdate": {
         "architecture": {
             "64bit": {
                 "url": "https://inkscape.org/en$matchWin64biturl",
-                "extract_dir": "$matchInkscapedir-x64"
+                "extract_dir": "$matchInkscapedir-x64",
+                "hash": {
+                    "url": "$matchWin32bithashurl"
```

Comment:
```suggestion
                    "url": "$matchWin32bithashurl",
                    "regex": "$sha256"
```

Reply:
I think there is something wrong *Scoop*'s hash matching machanism.
Therefore we need to **manually** match `$sha256` here. Will try to fix it later.
([related code](https://github.com/ScoopInstaller/Scoop/blob/master/lib/autoupdate.ps1#L70))